pkg/lib/cockpit-rsync-plugin.js

```diff
@@ -6,13 +6,17 @@ module.exports = class {
             options = {};
         this.dest = options.dest || "";
         this.source = options.source || "dist/";
+
+        // ensure the target directory exists
+        if (process.env.RSYNC)
+            child_process.spawnSync("ssh", [process.env.RSYNC, "mkdir", "-p", "/usr/local/share/cockpit/"], { stdio: "inherit" });
```

Comment:
This is ugly, but I didn't find an rsync option which would auto-create parent directories.

Reply:
There have been a few other times where I thought it might be useful to have `dist/` be called `cockpit/` or at least to make a symlink from `cockpit -> dist`, for exactly this reason.

Do you think that might help you here?